consensus/misc/eip6110.go

```diff
@@ -77,7 +80,7 @@ func ParseDepositLogs(logs []*types.Log, depositContractAddress libcommon.Addres
 	}
 	reqData := make([]byte, 0, len(logs)*types.DepositRequestDataLen)
 	for _, l := range logs {
-		if l.Address == depositContractAddress {
+		if l.Address == depositContractAddress && len(l.Topics) > 0 && l.Topics[0] == depositTopic {
```

Comment:
The deposit event might not be the first one: see https://sepolia.etherscan.io/tx/0xaebb84d80e12682fd1eafc494081cc8207abd26af3feb75c65b3c398561f2328#eventlog